src/browser/provider/built-in/index.js

```diff
@@ -0,0 +1,14 @@
+import PathBrowserProvider from './path';
+import LocalBrowserProvider from './local';
+import RemoteBrowserProvider from './remote';
+import saucelabsBrowserProvider from './saucelabs';
+import phantomJSBrowserProvider from './phantomjs';
+
+
+export default {
+    'local':     new LocalBrowserProvider(),
+    'path':      new PathBrowserProvider(),
```

Comment:
@inikulin @VasilyStrelyaev 
 Guys, I do not like the 'local' and  'path' names. For me, all browsers on my computer - local browsers.